src/chi-vue/src/components/data-table-tooltip/DataTableTooltip.tsx

```diff
@@ -0,0 +1,28 @@
+import { Component, Vue, Prop } from 'vue-property-decorator';
+import Tooltip from '@/components/tooltip/tooltip';
+import { UTILITY_CLASSES } from '@/constants/classes';
+
+@Component({})
+export default class DataTableTooltip extends Vue {
+  @Prop() msg!: string;
+
+  tooltip?: boolean = false;
+
+  setTooltip(state: boolean) {
+    this.tooltip = state;
+  }
+
+  render() {
+    return (
+      <div onMouseenter={() => this.setTooltip(true)}>
```

Comment:
Please, add `onMouseleave`to do the opposite as this injects Tooltip elements that currently don't get removed

https://user-images.githubusercontent.com/25136131/124724756-90fc7d80-df0c-11eb-99d5-751713db1234.mov

Reply:
hey @janibolkvadze it will be removed after `chitooltiphidden` event. If the you use `mouseleave` event it will be removed from dom before the Animation gets completed 

`onChiTooltipHidden={() => this.setTooltip(false)}` BTW I was investigating to see why randomly it doesn't get removed from DOM no clue yet

Reply:
FYI, I made some changes and it makes it happens less but still happening